docs/thirdparty.rst

```diff
@@ -30,12 +31,18 @@ The following modules are availble to patch::
         'aiobotocore',
         'boto3',
         'botocore',
+        'pynamodb',
         'requests',
         'sqlite3',
         'mysql',
     )
 
-Patching boto3 and botocore are equivalent since boto3 depends on botocore
+Patching boto3 and botocore are equivalent since boto3 depends on botocore.
+
+Patching pynamodb applies the botocore patch as well, as it uses the logic from the botocore
+patch to apply the trace header. As capturing the pynamodb calls relies on intercepting all
+calls made via botocore separately from the botocore patch, there is a small performance penalty
```

Comment:
I would not mention the `performance penalty` here. An extra if condition is really neglectable IMO considering how much code is actually executed in botocore.

Reply:
done